stdlib/3/pdb.pyi

```diff
@@ -1,5 +1,3 @@
-# Stub for pdb (incomplete, only some global functions)
```

Comment:
This comment still applies. It's accurate because the `pdb.PDB` class is missing from the stub.

Separately, this should be moved into 2and3.